### PR TITLE
Update the arm for safe operation 

### PR DESCRIPTION
**Pull request by jwt388**
These changes incorporate changes from issues 70 and 76, so those PRs need to be addressed first.

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -40,6 +41,8 @@ public void robotInit() {
     this.robotContainer = new RobotContainer();
 
     datalog.dataLogRobotContainerInit(this.robotContainer);
+
+    SmartDashboard.putData(CommandScheduler.getInstance());
```
Comment (blu28) [inline src/main/java/frc/robot/Robot.java:45]: Repeating comment from previous pull request. If possible, move this to into dataLogRobotContainerInit. If it isn't that is okay, but I would prefer to have datalogging isolated to datalogging.java, and putData calls in RobotContainer.java or in the subsystems.

**Review by blu28 — CHANGES_REQUESTED** [review]
Looks great. Only thought is to see about moving the new command scheduler putdata call into dataLoggingRobotContainerInit.

**Comment by jwt388** [comment]
I merged those changes into the Issue 82 branch.  Shuffleboard displays could use some formatting cleanup, but for now at least the command data is shown together. Other details of subsystem operation are still logged with SmartDashboard functions so the data goes to that general tab if auto-populate is enabled.  Otherwise, data can be displayed as needed for debug, and all data goes to the datalog file for post processing.

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (blu28) [reply src/main/java/frc/robot/Robot.java]: Since this seems to have been resolved in the other PR, I will resolve this convo.

**Review by blu28 — APPROVED** [review]

**Pull request merged**